GOVERNANCE.md

```diff
@@ -4,83 +4,139 @@ This document defines project governance for the Strimzi project.
+* Owners of components which have their own GitHub repository will get "Write" rights for given GitHub repository and will be able to merge approved PRs.
+* Owners will not get "Admin" rights on any Strimzi GitHub repositories.
 
-#### Github Project Administration
+Component owners can be removed by a vote.
+If the vote succeeds:
+* Their name will be removed from the `COMPONENT-OWNERS` file.
+* They will be removed to the collaborators list of the Strimzi repository, if any.
 
-Owners of components which have their own GitHub repository will get "Write" rights for given GitHub repository and will be able to merge approved PRs.
-Owners will not get "Admin" rights on any Strimzi GitHub repositories.
+Component owners who wish to become emeritus simply mark the fact by opening a PR against this repository moving their name to the `EMERITUS` file.
+The process for reverting from an emeritus component owners to an active component owners is the same as for new component owners.
 
-## Voting
+## Voting rules
 
 The Strimzi project employs voting to ensure no single member can dominate the project.
 
 For formal votes, a specific statement of what is being voted on should be added to the relevant GitHub issue or PR.
 Voters should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
 
-### Approving PRs
+The project uses different kinds of vote for different purposes.
+
+* PR approval votes
+* Lazy majority votes
+* Explicit majority votes
+
+Who may vote, where votes are recorded and the duration of the voting also depends on the subject of vote, as explained in the following sections.
 
-PRs may be merged after receiving at least two positive maintainer votes.
+### PR approval rules
+
+PRs against most repositories may be merged after receiving at least two positive maintainer votes.
 Voting for PRs can be done using a comment in the PR or by approving (or requesting changes) the PR.
 If the PR author is a maintainer, this counts as a vote.
 
-PRs which affect only a single component can be approved by at least one positive maintainer vote and one positive vote from an owner of given component.
+PRs which affect only a single component can be approved by at least one positive maintainer vote and one positive vote from an owner of the given component.
 If the PR author is a maintainer or an owner of given component, this counts as a vote.
 
-### Proposals
-
-Proposals can be opened against the [proposals repository](https://github.com/strimzi/proposals).
-Proposals should cover any changes to the Strimzi project which might significantly impact its users or the project direction.
+### Lazy majority rules
 
-Voting about proposals is using +1, 0, -1 votes and their fractions, where:
+Lazy majority votes use +1, 0, -1 votes and their fractions, where:
 * +1 means "yes"
 * -1 means "no"
 * The numbers between +1 and -1 indicate how strongly you feel about the proposal
```

Comment:
what does it mean? we can vote +0.4?

Reply:
Yes. This is in the current rules, so feel free to vote $+\pi^{-1}$ on this proposal if you want :stuck_out_tongue_winking_eye:

Reply:
I totally forgot about this rule. I should start voting in some more eccentric way :-D